polygeist/include/mlir/Dialect/Polygeist/IR/PolygeistDialect.h

```diff
@@ -1,16 +1,16 @@
-//===- Polygeist.h - Polygeist dialect --------------------------*- C++ -*-===//
+//===- PolygeistDialect.h - Polygeist dialect -------------------*- C++ -*-===//
 //
 // This file is licensed under the Apache License v2.0 with LLVM Exceptions.
 // See https://llvm.org/LICENSE.txt for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
 
-#ifndef MLIR_DIALECT_POLYGEIST_IR_POLYGEIST_H_
-#define MLIR_DIALECT_POLYGEIST_IR_POLYGEIST_H_
+#ifndef MLIR_DIALECT_POLYGEIST_IR_POLYGEISTDIALECT_H_
+#define MLIR_DIALECT_POLYGEIST_IR_POLYGEISTDIALECT_H_
```

Comment:
I'd rather have the trailing `_` removed (same for all cases) as we did in the `sycl` dialect.,

Reply:
done.